contracts/mapping-token/contracts/test/MockOutboundLane.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0;
+import "./MockMessageVerifier.sol";
+import "./MockInboundLane.sol";
+import "@darwinia/contracts-bridge/contracts/interfaces/IOnMessageDelivered.sol";
+
+contract MockOutboundLane is MockMessageVerifier {
+    struct ConfirmInfo {
+        address sender;
+        bool result;
+    }
+    address remoteInboundLane;
+    uint64 public nonce = 0;
+    mapping(uint64 => ConfirmInfo) responses;
+    constructor(
+        uint32 _thisChainPosition,
+        uint32 _thisLanePosition,
+        uint32 _bridgedChainPosition,
+        uint32 _bridgedLanePosition,
+        address _remoteInboundLane
+    ) public MockMessageVerifier(
+        _thisChainPosition,
+        _thisLanePosition,
+        _bridgedChainPosition,
+        _bridgedLanePosition
+    ) {
+        remoteInboundLane = _remoteInboundLane;
+    }
+
+    function send_message(address targetContract, bytes calldata encoded) external payable returns (uint64) {
```

Comment:
```suggestion
    function mock_dispatch(address targetContract, bytes calldata encoded) external payable returns (uint64) {
```

Reply:
this can't rename to mock_dispatch, because this func mock the actual `send_message` in message protocal's outbound.